files/build_templates/sonic_debian_extension.j2

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+FILESYSTEM_ROOT=$1
+[ -n "$FILESYSTEM_ROOT" ] || {
+    echo "Error: no or empty FILESYSTEM_ROOT argument"
+    exit 1
+}
```

Comment:
maybe simply saying load docker images and configurations

---
In reply to: [98135052](https://github.com/Azure/sonic-buildimage/pull/215#discussion_r98135052) [](ancestors = 98135052)